### ISSUE TITLE
Add validation for logLevel

### ISSUE THREAD

**Issue by yaelkeemink** [stryker-mutator/stryker-net#97]
At the moment you can give the CLI invalid loglevels which result in warning-loglevel. I think we might want to throw an error here.

**Comment by Valdas3** [comment]
I can take a look at this issue as my first thing for the hacktoberfest :)

**Comment by simondel** [comment]
Awesome! If you need any help, let us know :) It would be great if the validation is inside a generic part of stryker.net so it can also be used when we add support to run from visual studio (instead of using the CLI).

Do you agree with this @yaelkeemink?

**Comment by Valdas3** [comment]
I assume that by "generic part of stryker.net" you mean Stryker.Core project. 
In https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/README.md I found that default LogLevel should be error, hope it is up to date.
Also, I have updated the StrykerCLI_WithLogConsoleArgument_ShouldPassLogConsoleArgumentsToStryker unit test to include available log levels but it might make sense to move these somewhere else?
